### PR TITLE
Remove verbosity argument in scheduler

### PR DESCRIPTION
**Pull request by anwai98**
## What type of PR is this? (check all applicable)

- [x] Bug Fix
- [ ] Feature
- [ ] Optimization
- [ ] Documentation Update

## Description

With `pytorch` 2.7 onwards, `verbose` argument has been removed from learning rate schedulers. The trainings would break with new `pytorch` installation.

cc: @AlbertDominguez @maweigert

### CODE REVIEW

**Comment by AlbertDominguez** [comment]
Thanks a lot @anwai98! Merging rn
